Dockerfile.centos

```diff
@@ -74,6 +65,3 @@ RUN pip install -U cython
 
 # Install package
 RUN pip install /opt/file-metadata --upgrade
-
```

Comment:
We need some command to run in the end to test the successful install. At least an import test.